### PR TITLE
Adding comment on minikube about enabling metric pod

### PR DESCRIPTION
**Pull request by mmartin24**
Fixes https://github.com/epinio/docs/issues/204

### CODE REVIEW

**Review by enrichman — COMMENTED** [review]
Maybe we should say that the metric server is needed to get information about the status of the applications:

```
The metric server is required to get information about the status of the applications:
```

@andreas-kupries (phrasing help wanted)

**Comment by mmartin24** [comment]
> Maybe we should say that the metric server is needed to get information about the status of the applications:
> 
> ```
> The metric server is required to get information about the status of the applications:
> ```
> 
> @andreas-kupries (phrasing help wanted)

@enrichman, thanks taking your view and also @thehejik,  I would rephrase as:

```
It is recommended to enable a metric-server addon to get information about the status of the applications:
```

**Comment by mmartin24** [comment]
> 
> @enrichman, thanks taking your view and also @thehejik, I would rephrase as:
> 
> ```
> It is recommended to enable a metric-server addon to get information about the status of the applications:
> ```

I take this back. The metric-server takes more info that app status.
Changed to:

```
We recommend enabling the metrics-server addon to collect various statistics about resources:
```

**Review by thehejik — APPROVED** [review]
